supervisord/datadog_checks/supervisord/supervisord.py

```diff
@@ -115,11 +115,11 @@ def check(self, instance):
         # Filter monitored processes on configuration directives
         proc_regex = instance.get('proc_regex', [])
         if not isinstance(proc_regex, list):
-            raise Exception("Empty or invalid proc_regex.")
+            raise Exception("'proc_regex' should be a list of strings.")
```

Comment:
```suggestion
            raise Exception("'proc_regex' should be a list of strings: %s", proc_regex)
```
Printing out the value for `proc_regex` might be helpful for debugging?

Reply:
I don't know. They might think that the pattern itself is incorrect instead of the way they entered in their conf file.